vcpkg.json

```diff
@@ -1,6 +1,9 @@
 {
-    "dependencies": [ "gdal" ],
+    "name": "rasterio",
+    "version": "1.3.3",
+    "dependencies": [ "gdal:x64-windows" ],
+    "builtin-baseline": "57d3194e702a2959e86a6748999ad71fc24f7922",
     "overrides": [
-        { "name": "gdal", "version": "3.4.3#2" }
+        { "name": "proj:x64-windows", "version": "9.0.1" }
```

Comment:
You can specify the default triplet as an Env variable (see eg https://github.com/geopandas/pyogrio/blob/d1714041153416c0358658746a6b95f067c835a5/.github/workflows/release.yml#L165), so you don't need to specify it here (that causes the current error I think)

Reply:
And same for gdal above

Reply:
(it's also the default triplet, I assume, so your last commit to just remove it might indeed be sufficient)

Reply:
So far so good! :crossed_fingers: